CanvasAPI/CanvasAPI.py

```diff
@@ -71,12 +71,12 @@ def getOutcomeObject(self, outcomeID):
             outcomeObjectCount = len(outcomeObjects)
 
             if outcomeObjectCount == 0:
-                print 'log: successful response, but no outcome returned for ID: {}.' \
-                    .format(outcomeID)
+                print('log: successful response, but no outcome returned for ID: {}.' \
```

Comment:
why not use logging module to print instead of `print()`?

Reply:
Good idea.